php/libraries/UserPermissions.class.inc

```diff
@@ -163,6 +156,24 @@ class UserPermissions
         return true;
     }
 
+    /**
+     * Used to test whether a user has ANY of an array of specified permissions.
+     * Returns true on the first permission that the user has.
```

Comment:
The second sentence is unnecessary.

I think the behaviour of this is also wrong when passed an empty array. It should either be "true" (the empty set is a subset of any set) or an exception (it's probably a bug if called with an empty array)

Reply:
That's a good tip, thanks.